.travis.yml

```diff
@@ -84,8 +83,8 @@ before_script:
     - find ../project/instruments/NDB_BVL_Instrument_testtest.class.inc | php generate_tables_sql.php
     - cd ..
     - mysql LorisTest < project/tables_sql/testtest.sql
-#env:
- # - "LORIS_DB_CONFIG=test/config.xml"
+env:
+    - TRAVIS_CI=true LORIS_DB_CONFIG=/../test/config.xml
```

Comment:
I think we can just use one of the existing ones rather than adding another environment variable

https://docs.travis-ci.com/user/environment-variables#default-environment-variables

Reply:
done